js/table/tableview.ts

```diff
@@ -281,6 +290,15 @@ export class TableView {
         this.cardsVariantUp = ko.observable<boolean>(false);
         this.cardsVariantDown = ko.observable<boolean>(true);
 
+        this.roundNotification = ko.observable("");
+        this.roundNotificationTimer = ko.observable(0);
+        this.roundNotificationCaption = ko.computed(() => this.roundNotification() === "" ? "" : this.roundNotification());
```

Comment:
Why is such strange construct is even necessary is equivalent to `this.roundNotificationCaption = ko.computed(() => this.roundNotification());`.